platforms/android/gradle.properties

```diff
@@ -21,6 +21,8 @@ kotlin.code.style=official
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
 android.nonTransitiveRClass=true
+android.defaults.buildfeatures.buildconfig=true
+android.nonFinalResIds=false
```

Comment:
Defaults to true in AGP 8.
Do you really need this false? Are you relying on the actual value of some res id being const?

Reply:
That's the first and last time I'll use the AGP update assistant - I'd trusted that it would output sensible defaults but apparently not!